LISISO/commonfunctions.sh

```diff
@@ -315,3 +315,70 @@ function checkrpms()
 		echo "RPM's are missing"
 	fi
 }
+
+function CheckLibModulePartition()
+{
+        if (( $root_part_avail_space < $root_part_required_space ));then
+                echo "LIS installation aborted due to insufficient space"
+                printf "\t %s \t %s\n" "$lib_module_folder" "available: $root_part_avail_space required: $root_part_required_space"
+                return 0
+        fi
+        return 1
+}
+
+function CheckBootPartition()
+{
+        echo "boot part"
+        if (( $boot_part_avail_space < $boot_part_required_space ));then
+                echo "LIS installation aborted due to insufficiant space"
+                printf "\t %s \t\t %s\n" "$boot_folder" "available: $boot_part_avail_space required: $boot_part_required_space"
+                return 0
+        fi
+        return 1
+}
+
+function CheckRequiredSpace()
+{
+        No_of_symvers=`ls /boot |grep "symvers"|  wc -l`
+        MIN_SPACE_FOR_RAMFS_CREATION=167772160
+        lib_module_size_factor=1
+        if [ $No_of_symvers -gt 1 ];then
+                MIN_SPACE_FOR_RAMFS_CREATION=268435456
```

Comment:
Don't you want to handle symvers 3 or more?

Reply:
if No_of_symvers  is greater than 1(it can be any value >1), then MIN_SPACE_FOR_RAMFS_CREATION required is 268435456

If it is 1, "167772160" bytes is required.